giternity.py

```diff
@@ -254,6 +254,19 @@ def main():
                     url = repo["clone_url"]
                     metadata = gh_api.repo_to_cgitrc(repo)
                     archive_plan.append((repo, metadata, url, path))
+    else:
+        gh_api = GitHub(cgit_url=cgit_url, token='')
+    
+    # Arbitrary git repositories
+    arbitrary_repos = config.get("repos")
+    for repo in arbitrary_repos:
```

Comment:
This breaks when no config is given.

Reply:
Fixed here: https://github.com/arrdem/giternity/pull/1/files#diff-be3f87f4af411a082a372661b10d12590f2c10e06d77c138458ce537725d5cb1R262